std/assembly/typedarray.ts

```diff
@@ -83,6 +92,18 @@ export class Uint8ClampedArray extends Uint8Array {
   protected __unchecked_set(index: i32, value: i32): void {
     super.__unchecked_set(index, max(min(value, 255), 0));
   }
+
+  fill(value: u32, start: i32 = 0, end: i32 = i32.MAX_VALUE): Uint8ClampedArray {
+    return <Uint8ClampedArray>super.fill(value, start, end);
```

Comment:
should be `changetype<Uint8ClampedArray>(...)`

Reply:
Okay! Got it.

Reply:
```ts

  fill(value: u32, start: i32 = 0, end: i32 = i32.MAX_VALUE): Uint8ClampedArray {
    return changetype<Uint8ClampedArray>(super.fill(value, start, end));
  }

  sort(comparator: (a: u8, b: u8) => i32 = defaultComparator<u8>()): Uint8ClampedArray {
    return changetype<Uint8ClampedArray>(super.sort(comparator));
  }
```

This is implemented now.